discovery/srv.go

```diff
@@ -87,18 +87,18 @@ func srvFetch(cfg config.DiscoveryConfig) (*[]core.Backend, error) {
 
 			resp, err := srvDnsLookup(cfg, record.Target, dns.TypeA)
 			if err != nil {
-				log.Warn("Error fetching A record for ", record.Target, " skipping...")
+				log.Warn("Error fetching A record for ", record.Target, " skippinggithub.com/yyyar/gobetween.")
```

Comment:
Seems like incorrect ".." replacement, should remain "skipping..." :-)

Reply:
oh yeah, hehehe. Fixed.